ui/admin/app/components/form/credential-store/vault/index.hbs

```diff
@@ -85,119 +88,133 @@
       </InfoField>
     {{/if}}
   {{/if}}
-  <form.input
-    @name='address'
-    @type='text'
+
+  <Hds::Form::TextInput::Field
+    name='address'
     @value={{@model.address}}
-    @label={{t 'resources.credential-store.form.address.label'}}
-    @error={{@model.errors.address}}
-    @helperText={{t 'resources.credential-store.form.address.help'}}
-    disabled={{@model.isSaving}}
-    readonly={{false}}
-    as |field|
+    @isInvalid={{@model.errors.address}}
+    disabled={{form.disabled}}
+    {{on 'input' (set-from-event @model 'address')}}
+    as |F|
```

Comment:
non blocking question: im noticing that fields that are required don't have the "Required" badge and only "(Optional)" is added where needed. Is this a new pattern we should follow for forms?

Reply:
@lisbet-alvarez yes that is correct.  Thank you for the catch, incoming in next commit